### PR TITLE
Pia 2988 analysis ref docs 

### PR DESCRIPTION
**Pull request by llevente**

### CODE REVIEW

File: capture-sdk/sdk/src/main/java/net/gini/android/capture/analysis/AnalysisActivity.java [hunk 1]
```diff
@@ -46,18 +46,19 @@
...
- * instead by overriding the string resource named {@code gc_analysis_activity_indicator_message}. The message is displayed for images
- * only.
+ * instead by overriding the string resource named {@code gc_analysis_activity_indicator_message}.
  *
  * <p> For PDF documents the first page is shown (only on Android 5.0 Lollipop and newer) along with
- * the PDF's filename and number of pages above the page. On Android KitKat and older only the PDF's filename is shown with the preview area
+ * the PDF's filename. On Android KitKat and older only the PDF's filename is shown with the preview area
  * left empty.
  *
+ * <p> For images only the activity indicator and activity indicator message is shown without preview
```
Comment (zladzeyka) [inline capture-sdk/sdk/src/main/java/net/gini/android/capture/analysis/AnalysisActivity.java:55]: [Suggestion] Maybe `.` is missing?

File: capture-sdk/sdk/src/main/java/net/gini/android/capture/analysis/AnalysisHintsAnimator.java [hunk 2]
```diff
@@ -19,10 +19,7 @@
 import androidx.core.view.ViewPropertyAnimatorListenerAdapter;
 
 /**
- * Created by Alpar Szotyori on 09.05.2019.
```
Comment (zladzeyka) [inline capture-sdk/sdk/src/main/java/net/gini/android/capture/analysis/AnalysisHintsAnimator.java:22]: [Clarification] Did you agree with Alpar to remove copyright and keep only `Internal use only`?

**Review by zladzeyka — CHANGES_REQUESTED** [review]
Looks good 👍 😊
I have a few tiny notes only.

File: capture-sdk/sdk/src/main/java/net/gini/android/capture/analysis/AnalysisActivity.java [hunk 3]
```diff
@@ -46,18 +46,19 @@
  * document and an activity indicator while the document is being analyzed by the Gini API.
  *
  * <p> <b>Note:</b> The title from the ActionBar was removed. Use the activity indicator message
- * instead by overriding the string resource named {@code gc_analysis_activity_indicator_message}. The message is displayed for images
- * only.
+ * instead by overriding the string resource named {@code gc_analysis_activity_indicator_message}.
```
Comment (a-szotyori) [inline capture-sdk/sdk/src/main/java/net/gini/android/capture/analysis/AnalysisActivity.java:49]: [Request] You can replace this whole javadoc for AnalysisActivity with `Internal use only`. It's not part of the public API anymore.

Sorry that I wasn't able to tell you that earlier. 🙈

File: capture-sdk/sdk/src/main/java/net/gini/android/capture/analysis/AnalysisScreenPresenter.java [hunk 4]
```diff
@@ -57,9 +57,7 @@
 import static net.gini.android.capture.tracking.EventTrackingHelper.trackAnalysisScreenEvent;
 
 /**
- * Created by Alpar Szotyori on 08.05.2019.
- * <p>
- * Copyright (c) 2019 Gini GmbH.
+ * Internal use only
```
Comment (a-szotyori) [inline capture-sdk/sdk/src/main/java/net/gini/android/capture/analysis/AnalysisScreenPresenter.java:60]: [Request] We should keep the copyright section and add the `internal use only` below it in a separate javadoc like you did for AnalysisHintsAnimator.

**Review by a-szotyori — CHANGES_REQUESTED** [review]
Thank you and there are some minor things only.

**Review by a-szotyori — APPROVED** [review]

**Pull request merged**